spec/controllers/products_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe ProductsController, type: :controller do
+
+  describe 'GET #edit'do
+
+    let(:product) { create(:product) }
+      before { get :edit, params: { id: product } }
+      xit { expect (assigns(:product)).to eq(product) }
```

Comment:
oops, small typo..

Reply:
nice specs 👍

Reply:
xit just pend the code, I just got it working already